.github/workflows/ReleaseBlockBlobReader.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
             # If so we'll base our logic just on the last part.
             BRANCH=${BRANCH##*/}
             # Filter out illegal characters, replace with '-'
-            BRANCH=$(echo $BRANCH | tr '[:punct:]' _)
+            BRANCH=$(echo $BRANCH | tr '[:punct:]' -)
```

Comment:
Dependabot PRs are not building due to the calculated version not being a valid semantic version as the pre-release component contains underscores.

From https://semver.org/:
> A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92, 1.0.0-x-y-z.--.